### PR TITLE
FAQ menu hint changed

### PR DESCRIPTION
**Pull request by Crystalskulls**
Fix for Issue #2604

Changed the FAQ menu hint for small devices.

### CODE REVIEW

**Review by Ein-Tim — CHANGES_REQUESTED** [review]
This is not an optimal change as the text is also displayed if the menu is already open (e.g. on a desktop browser).

I suggest to find a better solution.

**Comment by MikeMcC399** [comment]
I was (again) writing in parallel to @Ein-Tim.

Anyway here is the screenshot. I think it is clear now that it needs some extra logic.

image

**Comment by Crystalskulls** [comment]
@Ein-Tim @MikeMcC399 I have committed some additional changes. The hint changes now, while resizing the window.

**Comment by MikeMcC399** [comment]
@Crystalskulls 
Excellent! 👍🏻

**Review by Ein-Tim — APPROVED** [review]
Works flawlessly! Thanks!

File: src/partials/page-faq-results.html [hunk 1]
```diff
@@ -39,7 +39,8 @@
         <div class="mb-2 col-12 col-md-2">
         <button id="faq-submit" value="search" class="btn btn-primary col-12">{{page-contents.section-main.texts.search-button}}</button>
       </div>
-        <div class="filterUse"><img src="../../../assets/img/icons/info.svg"><span>{{page-contents.section-main.texts.info-filteruse}}</span></div>
+        <div class="filterUse filterUse-md"><img src="../../../assets/img/icons/info.svg"><span>{{page-contents.section-main.texts.info-filteruse-md}}</span></div>
+        <div class="filterUse filterUse-sm"><img src="../../../assets/img/icons/info.svg"><span>{{page-contents.section-main.texts.info-filteruse-sm}}</span></div>
```
[the remark this answers is not in the document]
Comment (Crystalskulls) [reply src/partials/page-faq-results.html:43]: That's a good point. I will make an additional change.

**Review by larswmh — APPROVED** [review]

**Review by dsarkar — APPROVED** [review]

**Comment by dsarkar** [comment]
@Crystalskulls @MikeMcC399 @Ein-Tim Great! Thanks a lot for the PR and reviewing!

**Pull request merged**